src/Card/CardSelect.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import Form from '../Form';
+
+const CardSelect = ({
+  className,
+  cardsData,
+  onChange,
+  selectedCardName,
+}) => {
+  const [currentRecord, setCurrentRecord] = useState(null);
+  useEffect(() => {
+    setCurrentRecord(selectedCardName);
+  }, []);
```

Comment:
I think except for the empty array its dependency will be `selectedCardName`.

Reply:
This needs to run only once